### PR TITLE
issue 1128 fixed

### PR DESCRIPTION
**Pull request by vvianle**
Issue #1128 Remove "back" link on organization and project wrapper

### Proposed changes in this pull request

- Remove back/return to index link on organization wrapper and project wrapper.
- Increased font size of organization link on project wrapper by 2 pixels.

#### Functionality

- [x] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [x] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled. 

#### Code

- [x] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.

### CODE REVIEW

File: cadasta/templates/organization/project_wrapper.html [hunk 1]
```diff
@@ -13,9 +13,9 @@
 <!-- Single project header -->
 <div class="header page-header">
   <div class="page-title">
-    <a class="index-link visible-md-inline visible-lg-inline" href="{% url 'project:list' %}">
+<!--     <a class="index-link visible-md-inline visible-lg-inline" href="{% url 'project:list' %}">
```
Comment (oliverroick) [inline cadasta/templates/organization/project_wrapper.html]: Instead of commenting out the code, I would suggest to fully remove those lines from the template. We have version control so we can revert the changes if we ever decide to do so.

File: cadasta/templates/organization/organization_wrapper.html [hunk 2]
```diff
@@ -15,9 +15,9 @@
 <!-- Single organization header -->
 <div class="header page-header">
   <div class="page-title">
-    <a class="index-link visible-md-inline visible-lg-inline" href="{% url 'organization:list' %}">
+<!--     <a class="index-link visible-md-inline visible-lg-inline" href="{% url 'organization:list' %}">
```
Comment (oliverroick) [inline cadasta/templates/organization/organization_wrapper.html]: Instead of commenting out the code, I would suggest to fully remove those lines from the template. We have version control so we can revert the changes if we ever decide to do so.

**Comment by clash99** [comment]
Hi @vvianle, 

One more small change - Without the return link, we can tighten up that space to the left of the title. Can you adjust the  margin-left: 54px of the following tag to be 24px?  

#project-single .page-header h1, #project-single .page-header h2, #organization-single .page-header h1, #organization-single .page-header h2

**Comment by vvianle** [comment]
Thank you for your feedbacks! @clash99 @oliverroick

**Comment by oliverroick** [comment]
@clash99 Can you have a final look at the latest changes? If you approve we can move forward with this.
